src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -44,92 +129,175 @@ const ApplicationModal = props => {
         </Modal>
       ) : (
         <Modal
-          title="Application"
+          title="Review Application"
           visible={displayModal}
           onOk={handleOk}
           onCancel={handleCancel}
           afterClose={handleCancel}
           className="modalStyle"
-          footer={null}
+          footer={[
+            <Button key="back" onClick={handleCancel}>
+              Return to Previous
+            </Button>,
+            /**
+             * Author: Khaleel Musleh
+             * @param {onSubmit={approveApplication}} e Added an onSubmit Handler once the admin presses the approve button.
+             */
+            <Button key="submit" type="primary" onSubmit={approveApplication}>
```

Comment:
Hey Khaleel, for ant design, we need to use onConfirm instead of onSubmit here to be able to run the approveApplication once we click the confirm button from the pop up that says are you sure you want to approve this application?